Sources/Core/FoundationHTTP/Web3HttpProvider.swift

```diff
@@ -27,10 +27,13 @@ public struct Web3HttpProvider: Web3Provider {
     ]
 
     public let rpcURL: String
+    private let defaultRetryInterval: TimeInterval = 2
```

Comment:
It's better to set not equal intervals to reduce simultaneous requests from clients

Reply:
great idea! let's do that on a next improvement iteration. we would also need to the same for our internal networking layer.